### DISCUSSION TITLE
期望添加的新功能？

### DISCUSSION THREAD

**Discussion by Passer1072** [Passer1072/RookieAI_yolov8#16] [Ideas]
目前计划中功能

- [ ] 1. 自定义触发按键
- [x] 2. 垂直压枪
- [ ] 3. 可选择推理设备
- [x] 4. x轴偏移添加GUI
- [x] 5. 贝塞尔曲线鼠标平滑
- [ ] 6. 强化单目标锁定
- [x] 7. 随机瞄准位置（Y）
- [x] 8. 添加自动扳机UI
- [x] 9. 整合可选择鼠标移动库
- [ ] 10.使用PyQt重构UI
- [ ] 11.分离UI与主程序
- [x] 12.整合鼠标移动方式选择
- [ ]  13.可自定义配置飞易来PID VID
- [ ]  14.完善自动扳机功能

**Comment by DRElll** [comment]
能对接下kmboxnet后实现双机使用吗
